rest/config_database.go

```diff
@@ -90,6 +90,12 @@ func MergeDatabaseConfigWithDefaults(sc *StartupConfig, dbConfig *DbConfig) (*Db
 // to provide defaults to  include_runtime config endpoints.
 // Note that this does not include unsupported options
 func DefaultDbConfig(sc *StartupConfig) *DbConfig {
+	var partitions *uint16
+	if base.IsEnterpriseEdition() {
+		partitions = base.Uint16Ptr(base.GetDefaultImportPartitions(sc.IsServerless()))
+	} else {
+		partitions = nil
+	}
```

Comment:
```suggestion
	}
```

this is equivalent to declaring with `var` above